packages/main/src/themes/Button.css

```diff
@@ -244,53 +236,56 @@ bdi {
 :host([design="Emphasized"]:not([active]):not([non-interactive]):not([_is-touch]).ui5_hovered) {
 	background-color: var(--sapButton_Emphasized_Hover_Background);
 	border-color: var(--sapButton_Emphasized_Hover_BorderColor);
-	box-shadow: var(--sapContent_Informative_Shadow);
+	border-width: var(--_ui5_button_emphasized_border_width);
+	color: var(--sapButton_Emphasized_Hover_TextColor);
+	box-shadow: none;
 }
 
 :host([ui5-button][design="Empasized"][active]:not([non-interactive])) {
 	background-color: var(--sapButton_Emphasized_Active_Background);
 	border-color: var(--sapButton_Emphasized_Active_BorderColor);
 	color: var(--sapButton_Emphasized_Active_TextColor);
-	text-shadow: none;
 }
 
 :host([design="Emphasized"][focused]) .ui5-button-root:after {
-	content: "";
-	position: absolute;
-	box-sizing: border-box;
-	left: 0.0625rem;
-	top: 0.0625rem;
-	right: 0.0625rem;
-	bottom: 0.0625rem;
-	border: var(--_ui5_button_emphasized_focused_border);
-	pointer-events: none;
-	border-radius: var(--_ui5_button_emphasized_focused_border_radius);
+	border-color: var(--sapContent_ContrastFocusColor);
+	outline: none;
 }
 
 :host([design="Transparent"]) {
 	background-color: var(--sapButton_Lite_Background);
 	color: var(--sapButton_Lite_TextColor);
-	text-shadow: var(--_ui5_button_text_shadow);
 	border-color: var(--sapButton_Lite_BorderColor);
 }
 
-:host([design="Transparent"]:not([active]):not([non-interactive]):not([_is-touch]):hover) {
-	background-color: var(--_ui5_button_transparent_hover);
-}
-
 /*The ui5_hovered class is set by FileUploader to indicate hover state of the control*/
+:host([design="Transparent"]:not([active]):not([non-interactive]):not([_is-touch]):hover),
 :host([design="Transparent"]:not([active]):not([non-interactive]):not([_is-touch]).ui5_hovered) {
-	background-color: var(--_ui5_button_transparent_hover);
+	background-color: var(--sapButton_Lite_Hover_Background);
 	border-color: var(--sapButton_Lite_Hover_BorderColor);
-	box-shadow: var(--sapContent_Critical_Shadow)
+	box-shadow: var(--sapContent_Interaction_Shadow);
+	color: var(--sapButton_Lite_Hover_TextColor);
 }
 
 :host([ui5-button][design="Transparent"][active]:not([non-interactive])) {
-	background-color: var(--sapButton_Active_Background);
+	background-color: var(--sapButton_Lite_Active_Background);
+	border-color: var(--sapButton_Lite_Active_BorderColor);
 	color: var(--sapButton_Active_TextColor);
-	text-shadow: none;
 }
 
-:host([design="Transparent"]:not([active]):not([_is-touch]):hover) {
-	border-color: var(--sapButton_Lite_Hover_BorderColor);
+/* SegmentedButton and ToggleButton */
```

Comment:
Is there a reason to add the following styles are added here?
The Button styles are too many to also add styles related to descendants.

Please add them in the respective css files of the components. You can also introduce ui5-togglr-btn-root and ui5-segmented-btn-root (or similar) to distinguish and detach from ui5-button-root.
In this case, it's better to repeat them (if we see that the SegmentedButton and ToggleButton share a lot of common css that we write specifically for them, we can extract these styles to separate file for reuse).